tests/Sabre/Xml/ParserTest.php

```diff
@@ -0,0 +1,587 @@
+<?php
+
+namespace Sabre\Xml;
+
+/**
+ * Test the XML reader.
+ * -Test with valid document
+ * -Test XML schema validation
+ * -Test invalid documents (missing start and end tag, malformed start and end tag and malformed quotes)
+ *
+ *
+ * PHP version 5
```

Comment:
docblocks should match sabre/dav's docblocks, or be removed (we don't require them in unittests).